packages/rancher-operator-crd/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,12 @@
+--- charts-original/Chart.yaml
```

Comment:
FYI @aiyengar2 this will go into 2.5 while 2.6 has the master version, so we will see if they conflict once we merge

Reply:
Sounds good. This looks okay to me since they have different major.minor.patch versions; it's identical (from a scripts perspective) to having 2 packages that track the same chart at different major.minor.patch versions.